File: 백준/Bronze/2562. 최댓값/최댓값.c
int main()
{
    int a[9];

    for(int i=0; i<9; i++){
        scanf("%d\n", &a[i]);
    }

    int max = a[0];
    int n = 0;

    for(int j=0; j<9; j++){
        if(a[j] >= max){
            max = a[j];
            n = j;
        }
    }

    printf("%d\n", max);
    printf("%d\n", n+1);

    return 0;
}

File: 백준/Bronze/2562. 최댓값/최댓값.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString();

let arr = input.split("\n").map(Number);

let max = arr[0];
let idx = 0;

for (let i = 0; i < 9; i++) {
  max = Math.max(max, arr[i])
  if (max == arr[i]) 
    idx = i + 1;
}

console.log(max + "\n" + idx);